docs/data/date-pickers/custom-field/custom-field.md

```diff
@@ -49,6 +49,12 @@ You can manually add an `endAdornment` if you want your range picker to look exa
 This adornment is purely decorative, the focus remains on the field when the picker is opened.
 :::
 
+### Change the separator of range fields [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
```

Comment:
I moved this one below the introduction of the single input range fields since it uses them